operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java

```diff
@@ -222,7 +222,7 @@ public void testHostnameStrict() {
                         var curlOutput = K8sUtils.inClusterCurl(k8sclient, namespace, "-s", "--insecure", "-H", "Host: foo.bar", url);
                         Log.info("Curl Output: " + curlOutput);
 
-                        assertTrue(curlOutput.contains("var authServerUrl = 'https://example.com:8443';"));
+                        assertTrue(curlOutput.contains("var authServerUrl = 'https://example.com';"));
```

Comment:
Those changes are pretty much counter-intuitive since we are explicitly accessing Keycloak using the specific port:

https://github.com/keycloak/keycloak/blob/9ef3c289290b96d8e5cc7d7651296ba535b223a4/operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java#L219

This is caused by the fact that we are now setting `proxy: edge` and it means that we have to choose one of (but not the two):
 - make it work through the `Ingress`
 - make it work through straight port-forward
 
 Here we are biased toward access through the `Ingress` but this has drawbacks, especially for `local` and `developer` setups.
 
 cc. @pedroigor @DGuhr

Reply:
Could you elaborate what are the drawbacks? Don't you always access the instances through the ingress?

Reply:
> Don't you always access the instances through the ingress?

Not really, when deployed to K8s using the operator, at the moment, Keycloak can be accessed in multiple ways:
 - through the `Ingress`, basically using a proxy, used in production for user access mainly
 - through the `Service`, any k8s "internal" system will access Kc this way, direct access
 - using a `port-forward`, mostly for development/debugging purposes, re-binding remote ports to the machine's local
 
The expectation is that Keycloak is accessible and behaves the same through the 3 accesses unless something specific to prevent this behavior has been set.

The reality is that the access to the Admin console seems to be always not working in one of the 3 scenarios.

Reply:
I agree with this @DGuhr suggestion BTW:
> But perhaps we should open up a follow-up discussion/issue.